content/en/post/mudang-pdcat2023/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Mr. Soratouch's research presentation at PDCAT 2023"
+authors: ["soratouch-pornmaneerattanatri"]
+date: 2023-09-07T18:00:00+09:00
+featured: false
+draft: false
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder.
+# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
+image:
+  caption: ""
+  focal_point: ""
+  preview_only: false
+
+# Projects (optional).
+#   Associate this post with one or more of your projects.
+#   Simply enter your project's folder or file name without extension.
+#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
+#   Otherwise, set `projects = []`.
+projects: []
+
+---
+
+Mr. Soratouch Pornmaneerattanatri, a second-year doctoral student at SDLab presented the following paper at the [24th International Conference on Parallel and Distributed Computing, Applications and Technologies (PDCAT2023)](http://kips-cswrg.org/pdcat2023/)
+
+<!--more-->
+
+> Soratouch Pornmaneerattanatri, Keichi Takahashi, Yutaro Kashiwa, Kohei Ichikawa, Hajimu Iida, "Parallelizable Loop Detection using Pre-trained Transformer Models for Code Understanding", 24th International Conference on Parallel and Distributed Computing, Applications and Technologies (PDCAT2023), August. 2023.
+
+In this paper, we proposed a alternative methodology for automatic parallelization tool for classify parallelizable For-loop using deep learning based natualral language processing model. In this approach, we improve the accuracy of the automatic parallelization tools to identify the parallelizable For-loop by fine-tuned CodeT5 model compared to traditional approach, static analysis.
```

Comment:
->
In this paper, we proposed an alternative methodology for automatic parallelization tools to classify parallelizable For-loops using a deep learning-based natural language processing model. In this approach, we improved the accuracy of the automatic parallelization tool in identifying parallelizable For-loops compared to the traditional static analysis approach by a fine-tuned CodeT5 model.